client/flutter/pubspec.yaml

```diff
@@ -60,3 +61,10 @@ flutter:
 
 flutter_intl:
   enabled: true
+
+# To update the splash screen run the following command:
+# flutter pub pub run flutter_native_splash:create
```

Comment:
Is this right?

Reply:
Ah, I copied the command from the package's readme and it works but I do see the typo.